### ISSUE TITLE
Add bibtex entry to config

### ISSUE THREAD

**Issue by RichardLitt** [sustainers/academic-map#84]
See _config.yml comments.

**Comment by Nolski** [comment]
Do we have existing citations we want to list? If so, where are they stored?

**Comment by RichardLitt** [comment]
Yeah, we don't. I think I just saw this in the config and went "interesting"! 

Might be good to collect papers about the ecosystem, though. Like the EOSS papers.
